README.md

```diff
@@ -6,23 +6,62 @@ VSCode extension for replicating Grant Sanderson's `manim` workflow for Sublime
 
 ## Example usage
 
-After installing:
+1. [Install manimgl](https://3b1b.github.io/manim/getting_started/installation.html), make sure that the command `manimgl` works in your terminal
+2. Open your manim file (e.g. `scene.py`) in VSCode
+3. Place your cursor where you want to start the scene, on a line of code within the `construct` method in a scene class
+4. Run the VSCode command: `cmd+shift+p` -> `Manim Notebook: Start scene at cursor`  
+    (This command runs `manimgl scene.py NAME_OF_SCENE -se <lineNumber>` in your terminal)  
+    In the upper-right of your screen - the manim interactive video will appear.
 
-1. Open your manim `scene.py` in VSCode (replace `scene.py` with your manim file)
-2. In the attached terminal, run `manimgl scene.py NAME_OF_SCENE -se 155` (replace 155 with line number to start debugging). In the upper-right of your screen the manim interactive video should pop up
-3. Highlight the "section" you want to run (In manim, a section is a comment (`id`) followed by a block of code )
-4. Type `cmd+shift+p` "Checkpoint paste from manim-notebook" ( or default binding `cmd+' cmd+r` ). This will copy the selected code to the clipboard and send the `checkpoint_paste()` command from manim into the interactive window.
+<br />
+
+Then you can do either:
+
+- Write some comments which start with: `##`.  
+    The clickable "Preview Manim Cell" buttons will appear above each such comment.  
+    Clicking on one will run it.  
+    It is equivalent to: `cmd+shift+p` -> `Manim Notebook: Preview active Manim Cell`
+
+<br />
```

Comment:
This linebreak looks a bit odd to me in the Readme (since all other bullet points have the same spacing between them except this one). But leave it in if you like it ;)